frontend/insurance/src/app/core/components/navigation-bar/navigation-bar.component.spec.ts

```diff
@@ -22,4 +33,10 @@ describe('NavigationBarComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
```

Comment:
What is the point of this?

Reply:
Anytime you're expecting `componentX` to be truthy, it's basically a test for the component to be successfully created. Boilerplate provided by the CLI.